Update community-ecosystem-apps.md (#1713)
SumoLogic/sumologic-documentation: docs/integrations/community-ecosystem-apps.md

Community content is stored in our [Community Content Repository](https://github.com/SumoLogic/sumologic-content). Please visit the following links to access related content:

:::note

> The overall ratings for content are listed on a scale of 0-5. Full ratings and comments can get found within the Community Content Repository.

Reviewer: ```suggestion
The overall ratings for content are listed on a scale of 0 to 5, with 5 being the highest. You can view detailed ratings and comments on the Community Content Repository.
```
